murmurhash.js

```diff
@@ -15,12 +11,12 @@
    * @author <a href="mailto:aappleby@gmail.com">Austin Appleby</a>
    * @see http://sites.google.com/site/murmurhash/
    *
-   * @param {Buffer} str ASCII only
+   * @param {Buffer | Uint8Array | string} str ASCII only
```

Comment:
Given this typeof check below, do we need to do any additional disambiguation between Buffer and uint8array?

Reply:
Ah I see; https://nodejs.org/api/util.html#util_class_util_textencoder

I'm less familiar with modern node, does this mean TextEncoder doesn't need to be explicitly required to be used ?

Reply:
I just saw that Buffer extends Uint8Array

Reply:
So do you not have to require TextEncoder in NodeJS? It's just available globally?

Reply:
from node 11 yes